### ISSUE TITLE
Linter mode "manual" throwing an exception

### ISSUE THREAD

**Issue by whw** [SublimeLinter/SublimeLinter#1451]
Context:
macOS 10.13.5
Sublime Text 3.1.1 Build 3176
SublimeLinter v4.6.4
SublimeLinter-flake8 v4.1.0

Background:
When adding the setting `"lint_mode": "manual"` to my settings for SublimeLinter, I see an error. All other settings ("background". "load_save", and "save") do not produce an error.

Settings:
```
// SublimeLinter Settings - User
{
	"debug": true,
	"lint_mode": "manual",
	"linters": {
		"flake8": {
			"executable": "~/miniconda3/bin/flake8",
			"show-code": false
		},
	},
}
```

Error (from SublimeText console):
```
SublimeLinter: linter.py:798: Checking lint mode manual vs lint reason None
Exception in thread Thread-44:
Traceback (most recent call last):
  File "./python3.3/threading.py", line 901, in _bootstrap_inner
  File "./python3.3/threading.py", line 1142, in run
  File "<home>/Library/Application Support/Sublime Text 3/Installed Packages/SublimeLinter.sublime-package/sublime_linter.py", line 257, in lint
    linter for linter in linters
  File "<home>/Library/Application Support/Sublime Text 3/Installed Packages/SublimeLinter.sublime-package/sublime_linter.py", line 258, in <listcomp>
    if linter.should_lint(reason)]
  File "<home>/Library/Application Support/Sublime Text 3/Installed Packages/SublimeLinter.sublime-package/lint/linter.py", line 801, in should_lint
    return reason in _ACCEPTABLE_REASONS_MAP[lint_mode]
TypeError: 'in <string>' requires string as left operand, not NoneType
```

**Comment by kaste** [comment]
Thanks for reporting! Fix is underway.

Also this is just sad! There was a comma missing, something a linter should handle and warn.

**Comment by whw** [comment]
Hahaha, oh the irony. Thanks for the fix!
